internal/testrunner/compiler_runner.go

```diff
@@ -89,7 +89,8 @@ func (r *CompilerBaselineRunner) RunTests(t *testing.T) {
 	r.cleanUpLocal(t)
 	files := r.EnumerateTestFiles()
 	skippedTests := []string{
-		"mappedTypeRecursiveInference.ts", // Needed until we have type printer with truncation limit.
+		"mappedTypeRecursiveInference.ts",         // Needed until we have type printer with truncation limit.
+		"jsFileCompilationWithoutJsExtensions.ts", // Needed until we have proper allowJS support (and errors when not enabled.)
```

Comment:
Does this crash, or just fail?

Reply:
It crashes due to a panic in DoJsEmitBaseline that we would want to keep. The issue is that we're not reporting a diagnostic related to `allowJS` that hasn't been implemented in program yet.